### PR TITLE
SDK-13 DOCS-11914 Document uses of parameters of type Scheduler in extensions

### PR DESCRIPTION
**Pull request by ndinu**

### CODE REVIEW

File: modules/ROOT/pages/define-parameters.adoc [hunk 1]
```diff
@@ -571,6 +571,11 @@ public class DbPoolingProfile implements DatabasePoolingProfile {
 }
 ----
 
+== Parameter Type Restrictions
+
+It is not allowed for developers to add parameters or POJOs with fields of types _org.mule.runtime.core.api.source.scheduler.Scheduler_ or _org.mule.runtime.api.scheduler.SchedulingStrategy_ . The Mule Runtime and the SDK will add parameters of these types, for example, in Pooling Sources.
```
Comment (SebaElizalde) [inline modules/ROOT/pages/define-parameters.adoc]: typo: polling sources

**Review by SebaElizalde — DISMISSED** [review]

File: modules/ROOT/pages/define-parameters.adoc [hunk 2]
```diff
@@ -571,6 +571,11 @@ public class DbPoolingProfile implements DatabasePoolingProfile {
 }
 ----
 
+== Parameter Type Restrictions
+
+You cannot add parameters or POJOs with types _org.mule.runtime.core.api.source.scheduler.Scheduler_ or _org.mule.runtime.api.scheduler.SchedulingStrategy_. Mule runtime and the SDK adds parameters of these types, for example, in polling sources.
```
Comment (hannanelson) [inline modules/ROOT/pages/define-parameters.adoc]:
```suggestion
You cannot add parameters or POJOs with types `org.mule.runtime.core.api.source.scheduler.Scheduler` or `org.mule.runtime.api.scheduler.SchedulingStrategy`. Mule runtime engine and the SDK add parameters of these types, for example, in polling sources.
```

File: modules/ROOT/pages/parameters.adoc [hunk 3]
```diff
@@ -298,4 +298,10 @@ There are some subtle differences between these two concepts. You should definit
...
 
-However, the `Options` POJO defined above concerns the object itself, not just the parameters it contains. For instance, it would make sense to have a list of `Options` because each object is something on its own (in the OOP sense). On the other hand, if you are using a class simply for the sake of grouping things together, without any particular meaning for your module's domain, then it makes sense to have a parameter group.
\ No newline at end of file
+However, the `Options` POJO defined above concerns the object itself, not just the parameters it contains. For instance, it would make sense to have a list of `Options` because each object is something on its own (in the OOP sense). On the other hand, if you are using a class simply for the sake of grouping things together, without any particular meaning for your module's domain, then it makes sense to have a parameter group.
+
+
+== Parameter Type Restrictions
+
+You cannot add parameters or POJOs with types _org.mule.runtime.core.api.source.scheduler.Scheduler_ or
```
Comment (hannanelson) [inline modules/ROOT/pages/parameters.adoc]:
```suggestion
You cannot add parameters or POJOs with types `org.mule.runtime.core.api.source.scheduler.Scheduler` or
```

File: modules/ROOT/pages/parameters.adoc [hunk 4]
```diff
@@ -298,4 +298,10 @@ There are some subtle differences between these two concepts. You should definit
...
+_org.mule.runtime.api.scheduler.SchedulingStrategy_.
```
Comment (hannanelson) [inline modules/ROOT/pages/parameters.adoc]:
```suggestion
`org.mule.runtime.api.scheduler.SchedulingStrategy`.
```

**Review by hannanelson — DISMISSED** [review]

**Review by hannanelson — APPROVED** [review]

**Pull request merged**